Dockerfile

```diff
@@ -1,7 +1,13 @@
 FROM ubuntu:16.04
 
-RUN apt-get update && apt-get install -y curl  libunwind8 libicu55
-RUN curl -LO https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.9/powershell_6.0.0-alpha.9-1ubuntu1.16.04.1_amd64.deb
-RUN dpkg -i powershell_6.0.0-alpha.9-1ubuntu1.16.04.1_amd64.deb
+ENV POWERSHELL_VERSION="6.0.0-alpha.9"
 
-Entrypoint [ "powershell" ]
+RUN apt-get update && \
+    apt-get install -y curl libunwind8 libicu55 && \
+    curl -LO https://github.com/PowerShell/PowerShell/releases/download/v"$POWERSHELL_VERSION"/powershell_"$POWERSHELL_VERSION"-1ubuntu1.16.04.1_amd64.deb && \
+    dpkg -i powershell_"$POWERSHELL_VERSION"-1ubuntu1.16.04.1_amd64.deb && \
```

Comment:
Do you have a fixed URL for the download script? I see it in tools in the source. Alternatively we could just explicitly add the download script from the source?

Another reason may be that the script always gets latest. Which the user has to know that and force an uncached build of the image to get the latest if one has been released.

The ENV approach at least makes everything explicit, with the downside it needs to be kept up to date. Easier once you have the mentioned package repo.